client/src/components/UserManagement.jsx

```diff
@@ -0,0 +1,201 @@
+import * as React from "react";
```

Comment:
Put this inside a user-management-component folder in a similar fashion as is done on the main branch. Also please add the admin dashboard under the pages directory(in src) like in the main branch.